src/api/bulk.ts

```diff
@@ -39,13 +44,46 @@ export type BulkOptions = {
 
 export type JobState = 'Open' | 'Closed' | 'Aborted' | 'Failed' | 'Unknown';
 
+export type JobStateV2 =
+  | Exclude<JobState, 'Closed' | 'Unknown'>
+  | 'UploadComplete'
+  | 'InProgress'
+  | 'JobComplete';
+
 export type JobInfo = {
   id: string;
   object: string;
   operation: BulkOperation;
   state: JobState;
 };
 
+export type JobInfoV2 = {
+  apiVersion: string;
+  assignmentRuleId?: string;
+  columnDelimiter:
+    | 'BACKQUOTE'
+    | 'CARET'
+    | 'COMMA'
+    | 'PIPE'
+    | 'SEMICOLON'
+    | 'TAB';
+  concurrencyMode: string;
```

Comment:
It seems like the only mode supported right now is "Parallel", should we just put that here so it can be an enum later when more modes are added?

(Similar to `contentType: 'CSV'` below)